eos-factory-reset.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Endless factory reset service
+ConditionPathExists=/var/eos-factory-reset
+Conflicts=getty@tty1.service gdm.service plymouth-start.service
+After=systemd-user-sessions.service getty@tty1.service plymouth-quit.service gdm.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/lib/eos-factory-reset/eos-factory-reset-helper
```

Comment:
I feel very strongly that the `.service` file should live in the same repository as the script it runs.